### PR TITLE
Added color.js build to code-studio so it doesn't depend on apps being built first

### PR DESCRIPTION
**Pull request by aoby**
Tested manually by building both apps and code-studio via `npm run build` after deleting color.js in both places.

### CODE REVIEW

**Comment by islemaster** [comment]
Have you tried `rake adhoc:start` yet?

**Comment by aoby** [comment]
Trying now. I'll update when it's done...

**Comment by islemaster** [comment]
LGTM.  One thought; I wonder if "shared/scripts/convertScssToJs.js" should be in "tools/scripts" instead, alongside our git hooks.  I'm fine either way.

**Comment by aoby** [comment]
Yeah I like that better. I'll move it to tools/scripts.

**Comment by aoby** [comment]
`rake adhoc:start` succeeded, and I moved convertScssToJs.js to tools/scripts.

File: code-studio/package.json [hunk 1]
```diff
@@ -39,7 +39,10 @@
     "build-css:dist": "npm run lint-css && npm run clean-css && scripts/build-css.js --dist",
     "build-js": "scripts/build-js.js",
     "build-js:dist": "npm run lint-js && npm test && npm run clean-js && scripts/build-js.js --dist && scripts/uglify-js.js",
-    "start": "scripts/build-js.js --watch"
+    "start": "scripts/build-js.js --watch",
+    "convert-scss-vars": "scripts/convert-scss-variables.js",
+    "prebuild-js": "npm run convert-scss-vars",
+    "prebuild-js:dist": "npm run convert-scss-vars"
```
Comment (Bjvanminnen) [inline code-studio/package.json:45]: I think we want to update our build scripts to also call prebuild-js?

Comment (islemaster) [inline code-studio/package.json:45]: They're automatically run before `build-js` thanks to [npm magic](https://docs.npmjs.com/misc/scripts):

> Additionally, arbitrary scripts can be executed by running npm run-script <pkg> <stage>. Pre and post commands with matching names will be run for those as well (e.g. premyscript, myscript, postmyscript).

**Pull request merged**